cmd/sysl/cmd_mod.go

```diff
@@ -0,0 +1,19 @@
+package main
```

Comment:
Lets not add this to main package and figure out a separate package for this.

Reply:
I think @Joshcarp is working on this(#478 ), so I just started as it is. I will sync up the code with his changes after his part's done.

Reply:
No.... main will continue to have these cmd files, they should just call into pkg/<something> to actually do the work. think of main as the frontend and pkg/ as the backend